recipes/gmp/all/conanfile.py

```diff
@@ -65,106 +61,105 @@ def config_options(self):
 
     def configure(self):
         if self.options.shared:
-            del self.options.fPIC
+            self.options.rm_safe("fPIC")
         if self.options.get_safe("enable_fat"):
             del self.options.disable_assembly
         if not self.options.enable_cxx:
-            del self.settings.compiler.libcxx
-            del self.settings.compiler.cppstd
+            self.settings.rm_safe("compiler.libcxx")
+            self.settings.rm_safe("compiler.cppstd")
 
-    def validate(self):
-        if self._is_msvc and self.options.shared:
-            raise ConanInvalidConfiguration("Cannot build a shared library using Visual Studio: some error occurs at link time")
+    def layout(self):
+        basic_layout(self, src_folder="src")
 
     def package_id(self):
         del self.info.options.run_checks  # run_checks doesn't affect package's ID
 
+    def validate(self):
+        if is_msvc(self) and self.options.shared:
+            raise ConanInvalidConfiguration(
+                f"{self.ref} cannot be built as a shared library using Visual Studio: some error occurs at link time",
+            )
+
     def build_requirements(self):
-        self.build_requires("m4/1.4.19")
-        if self._settings_build.os == "Windows" and not tools.get_env("CONAN_BASH_PATH"):
-            self.build_requires("msys2/cci.latest")
-        if self._is_msvc:
-            self.build_requires("yasm/1.3.0")
-            self.build_requires("automake/1.16.4")
+        self.tool_requires("m4/1.4.19")
+        if self._settings_build.os == "Windows":
+            self.win_bash = True
+            if not self.conf.get("tools.microsoft.bash:path", check_type=str):
+                self.tool_requires("msys2/cci.latest")
+        if is_msvc(self):
+            self.tool_requires("yasm/1.3.0")
+            self.tool_requires("automake/1.16.5")
 
     def source(self):
-        tools.get(**self.conan_data["sources"][self.version], destination=self._source_subfolder, strip_root=True)
+        get(self, **self.conan_data["sources"][self.version], strip_root=True, verify=False)
```

Comment:
verify false?! I am not even sure what that does 🤔 is the HTTPS on the source remote broken?

Reply:
> is the HTTPS on the source remote broken?

I guess, it fails without verify=False

Reply:
Certificate expired end of last year.